tests/test_dataloader.py

```diff
@@ -177,7 +177,7 @@ def test_denoising_dataloader(decoder_only_format, pretokenize, packing_ratio):
                 'sequence_mask_ratios': 0.25,
             },
             'drop_last': False,
-            'num_workers': 0,
+            'num_workers': 4,
```

Comment:
any reason you chose four?

Reply:
torch2 only allows prefetching if you have multiple workers (which makes sense)
I added workers instead of trying to strip out prefetching; could technically have gone either way.

@alextrott16 said it was ok (he wrote the test)